pinot-integration-tests/src/test/resources/compat-tests/queries/feature-test-1-sql.queries

```diff
@@ -1,89 +1,89 @@
 # Aggregation
-SELECT count(*) FROM FeatureTest1
-SELECT sum(intMetric1), sumMV(intDimMV1), min(intMetric1), minMV(intDimMV2), max(longDimSV1), maxMV(intDimMV1) FROM FeatureTest1
-SELECT count(longDimSV1), countMV(intDimMV1), avg(floatMetric1), avgMV(intDimMV2), minMaxRange(doubleMetric1), minMaxRangeMV(intDimMV2) FROM FeatureTest1
-SELECT percentile(longDimSV1, 80), percentileMV(intDimMV1, 90), percentileEst(longDimSV1, 80), percentileEstMV(intDimMV1, 90), percentileTDigest(longDimSV1, 80), percentileTDigestMV(intDimMV1, 90) FROM FeatureTest1
-SELECT distinctCount(longDimSV1), distinctCountMV(intDimMV1), distinctCountHLL(longDimSV1), distinctCountHLLMV(intDimMV1) FROM FeatureTest1
+SELECT count(*) FROM FeatureTest1 WHERE __GENERATIONNUMBER__ = GENERATION_NUMBER
```

Comment:
```suggestion
SELECT count(*) FROM FeatureTest1 WHERE generationNumber = __GENERATION_NUMBER__
```

Same for the other queries. The column name should be `generationNumber`. The string that you sbstitute can be `__GENERATION+NUMBER__`